nntrainer/src/conv2d_layer.cpp

```diff
@@ -66,11 +66,15 @@ int Conv2DLayer::initialize(bool last) {
 }
 
 void Conv2DLayer::read(std::ifstream &file) {
-  // NYI
+  std::for_each(filters.begin(), filters.end(),
+                [&](Tensor i) { i.read(file); });
```

Comment:
Isn't there copying `Tensor` happening with this signature? (same for line 71~77)
```suggestion
                [&](Tensor &i) { i.read(file); });
```

Reply:
right. fixed. Thanks.